src/utils.ts

```diff
@@ -1,21 +1,32 @@
 import imageUrlBuilder from '@sanity/image-url';
 
-import { SanityClientLike, SanityImageObject } from './types';
+import { SanityClientLike, SanityImageObject, ImageUrlBuilderOptionsWithAliases, BuilderOptions } from './types';
 
 /**
  * Various widths at which to generate image sources in `srcset`. First item is used as the default width in fallback `img` elements.
  */
-export const SOURCE_WIDTHS = [320, 640, 960, 1280, 1600, 1920, 2240];
+export const DEFAULT_SOURCE_WIDTHS = [320, 640, 960, 1280, 1600, 1920, 2240];
 
 /**
- * Get dimensions of the specified image at the default size, which is a `width` of the first element in `SOURCE_WIDTHS` and a corresponding `height` based on cropped or default aspect ratio, overrideable with the `aspectRatio` argument
+ * The fallback quality that will be used when none is provided.
+ */
+export const DEFAULT_FALLBACK_IMAGE_QUALITY = 90;
```

Comment:
Just a minor thing but wondering if `DEFAULT_IMAGE_QUALITY` might be a better name for this const as `DEFAULT_FALLBACK` seems like a tautology.